README.md

```diff
@@ -38,74 +36,86 @@ The example snippets below show how to acquire a token for Microsoft Graph.
 
 ```JavaScript
-   var graphScopes = ["user.read", "mail.send"];
+   var loginRequest = {
+       scopes: ["user.read", "mail.send"] // optional
+   };
 
-   userAgentApplication.loginPopup(graphScopes).then(function (idToken) {
-       //login success
-   }, function (error) {
-       //login failure
-       console.log(error);
-   });
+    msalInstance.loginPopup(loginRequest)
+        .then(response => {
+            // handle response
+        })
+        .catch(err => {
+            // handle error
+        });
 
 ```
 > Note: The scopes passed to the login method are optional. In this example, the graphScopes are passed in the login method to obtain consent upfront from the user for your app to access certain Graph API scopes. The idtoken returned here does not contain the scopes. In the next step, you can see how to get an access token which will contain the consented scopes.
 
 #### 3. Get an access token to call an API
 
-In MSAL, you can get access tokens for the APIs your app needs to call using the acquireTokenSilent method which makes a silent request(without prompting the user) to Azure AD to obtain an access token. The Azure AD service then returns an [access token](https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens) containing the user consented scopes to allow your app to securely call the API.  
+In MSAL, you can get access tokens for the APIs your app needs to call using the acquireTokenSilent method which makes a silent request(without prompting the user) to Azure AD to obtain an access token. The Azure AD service then returns an [access token](https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens) containing the user consented scopes to allow your app to securely call the API.
 
 If the silent token acquisition fails for some reasons such as an expired token or password change, you will need to invoke an interactive method to acquire tokens such as acquireTokenPopup or acquireTokenRedirect.
 
  ```JavaScript
-    var graphScopes = ["user.read", "mail.send"];
-
-    userAgentApplication.loginPopup(graphScopes).then(function (idToken) {
-        //Login Success
-        userAgentApplication.acquireTokenSilent(graphScopes).then(function (accessToken) {
-            //AcquireTokenSilent Success
-        }, function (error) {
-            //AcquireTokenSilent Failure, send an interactive request.
-            userAgentApplication.acquireTokenPopup(graphScopes).then(function (accessToken) {
-                updateUI();
-            }, function (error) {
-                console.log(error);
-            });
+
+    var tokenRequest = {
+        scopes: ["user.read", "mail.send"]
+    };
+
+    msalInstance.loginPopup(loginRequest)
```

Comment:
where is loginRequest defined?

Reply:
removing and adding comment, that if you are not calling with scopes, you don't have to pass an obj